lib/models/Cat.js

```diff
@@ -0,0 +1,95 @@
+const pool = require('../utils/pool');
+
+module.exports = class Cat {
+  id;
+  name;
+  age;
+  type;
+
+  constructor(row) {
+    this.id = row.id;
+    this.name = row.name;
+    this.age = row.age;
+    this.type = row.type;
+  }
+
+  getHello() {
+    return `Meow I'm ${this.name} and I am ${this.age} years old`;
+  }
+
+  static async insert({ name, age, type }) {
+    try {
+      const { rows } = await pool.query(
+        'INSERT INTO cats(name, age, type) VALUES ($1, $2, $3) RETURNING *;',
+        [name, age, type]
+      );
+      return new Cat(rows[0]);
+    } catch (error) {
+      return null;
+    }
+  }
+
+  static async findALL() {
+    const { rows } = await pool.query(
+      `
+      SELECT
+        *
+      FROM 
+        cats
+      `
+    );
```

Comment:
We can return early here if nothing was returned from the query to prevent errors. Something like `if(!rows[0]) return null`